samples/ControlCatalog.NetCore/Program.cs

```diff
@@ -112,10 +120,12 @@ public static AppBuilder BuildAvaloniaApp()
                     EnableMultiTouch = true,
                     UseDBusMenu = true,
                     EnableIme = true,
+                    UseCompositor = true
                 })
                 .With(new Win32PlatformOptions
                 {
-                    EnableMultitouch = true
+                    EnableMultitouch = true,
+                    UseCompositor = true
```

Comment:
There's `UseCompositor` here and `UseComposition` above in `AvaloniaHeadlessPlatformOptions` - do these do the same thing? Should they have the same name?